source/retryable-reads/tests/aggregate-merge.yml

```diff
@@ -1,6 +1,7 @@
 runOn:
   -
     minServerVersion: "4.1.11"
+    serverless: "forbid"
```

Comment:
Is this missing a comment about why serverless is forbidden here? I assume it's because of fail points.